README.md

```diff
@@ -10,49 +10,31 @@
 [![Rating](https://img.shields.io/visual-studio-marketplace/r/WatermelonTools.watermelon-tools?style=flat-square)](https://marketplace.visualstudio.com/items?itemName=WatermelonTools.watermelon-tools&ssr=false#review-details)
 [![Discord](https://img.shields.io/discord/933846506438541492?style=flat-square)](https://discord.com/invite/H4AE6b9442)
 
-Watermelon is your **Open Source** Code Archeology Toolbox for **Visual Studio Code**. We allow developers to find the most relevant piece of context for a given block of code. We help you __grok__ codebases by indexing information from git, ticketing, and messaging systems. 
+**Watermelon helps your team merge Pull Requests With Business Value**. 
 
-## Free Features
+We've built a search algorithm that indexes the most relevant [passive documentation](https://www.watermelontools.com/post/what-is-passive-code-documentation-why-is-it-hard-to-scale-what-to-do-about-it) for a given block of code or pull request. 
 
-Watermelon has 2 features that you can use for free:
+To obtain business context in your IDE, use this extension.
 
-- Hover to get context
-- Get most relevant Pull Request 
-- Get the most relevant Jira Tickets
-- Get the most relevant Slack message thread
-
-More integrations coming soon!
-
-### Hover To Get Context
-
-The fastest way to use Watermelon is to hover over a line of code to get its context.
-
-In the image below you can see how hovering over a line of code provides us the following:
+To obtain business context in your CI/CD, take a look at our [GitHub Application](https://github.com/apps/watermelon-context)
 
-![hover](https://thumbs.gfycat.com/EcstaticSpryAracari-size_restricted.gif)
 
-- The author and date of the latest commit
-- The message of the latest commit
-- The number of times the current file has changed
-- A button that says "View the code context with Watermelon"
+## Integrations
 
-If you want to do a deeper research about the code context history, simply click such button. 
+We currently support the following integrations
 
-### Get Full Code Context
+|               | Git                                   | Project Management | Messaging | Documentation |
+| :------------ | :------------------------------------ | :----------------- | :-------- | :-----------  |
+| VS Code       | GitHub, GitLab, Bitbucket             | Jira               | Slack     |               |
+| GitHub App    | GitHub                                | Jira               | Slack     | Notion        |
```

Comment:
The extension currently does not fetch from Notion

Reply:
Correct. This is how the rendered table looks

<img width="950" alt="Screen Shot 2023-06-15 at 5 20 16 PM" src="https://github.com/watermelontools/watermelon-extension/assets/8325094/f692bcdf-4d6b-4c65-ad1f-c2b98b509cd2">